frontend/components/Login.vue

```diff
@@ -20,7 +20,7 @@
             </label>
             <input v-model="username"
                    class="appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none"
-                   id="username" type="text" placeholder="user@example.com" name="username" autofocus>
```

Comment:
Wat is de rede dat autofocus hier is weggehaald?

Reply:
Autofocus processing was blocked because a document's URL has a fragment '#/login'.